sonar-html-plugin/src/main/java/org/sonar/plugins/html/node/TagNode.java

```diff
@@ -102,16 +101,17 @@ public List<TagNode> getChildren() {
   }
 
   public String getLocalName() {
-    String localPart = StringUtils.substringAfterLast(getNodeName(), ":");
-    if (StringUtils.isEmpty(localPart)) {
-      return nodeName;
+    String[] parts = getNodeName().split(":");
+    String localPart = parts[parts.length - 1];
+    if (localPart.isEmpty()) {
+      return getNodeName();
     } else {
       return localPart;
     }
   }
 
   public String getNodeName() {
-    return nodeName;
+    return nodeName == null ? "" : nodeName;
```

Comment:
_minor_ Same as for `Node`, I'd would rather put this in the setter.

Reply:
(see comment about `setCode`, it s a similar reasoning)